project/src/test/java/com/google/sps/test/PlacesFetcherTest.java

```diff
@@ -119,17 +121,19 @@ private static PlacesSearchResult createTestPlacesSearchResult(String placeId) {
   @Test
   public void fetch_zeroSearchResults_returnsEmptyList() throws Exception {
     PlacesFetcher spiedFetcher = spy(placesFetcher);
-    doReturn(new PlacesSearchResult[0]).when(spiedFetcher).getPlacesSearchResults();
+    doReturn(new PlacesSearchResult[0])
+        .when(spiedFetcher).getPlacesSearchResults(any(TextSearchRequest.class));
     assertEquals(ImmutableList.of(), spiedFetcher.fetch());
   }
 
   @Test
   public void fetch_validSearchResults_returnsListOfPlaces()
```

Comment:
Yes, I really tried to do that for a while yesterday because it's bugging me that those methods are not really tested but I didn't find any way to get the info from the requests so I gave up for now.